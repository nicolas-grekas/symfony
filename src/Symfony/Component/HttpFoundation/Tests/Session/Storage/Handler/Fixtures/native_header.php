<?php

require __DIR__.'/common.inc';

$res = new \Symfony\Component\HttpFoundation\Response();
$res->headers->set('Set-Cookie', 'key=value; path=/; domain=example.org; HttpOnly; SameSite=Strict');
$res->headers->set('Date', 'Thu, 05 Apr 2018 12:12:12 GMT');
$res->sendHeaders();
