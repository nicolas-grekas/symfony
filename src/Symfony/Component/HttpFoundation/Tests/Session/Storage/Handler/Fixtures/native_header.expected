
Array
(
    [0] => Content-Type: text/plain; charset=utf-8
    [1] => Cache-Control: no-cache, private
    [2] => Date: Thu, 05 Apr 2018 12:12:12 GMT
    [3] => Set-Cookie: key=value; path=/; domain=example.org; httponly; samesite=strict
)
shutdown
